src/js/hca-rjsf/config/form.js

```diff
@@ -233,8 +236,40 @@ const formConfig = {
         }
       }
     },
+    vaBenefits: {
+      title: 'VA Benefits',
+      pages: {
+        vaBenefits: {
+          path: 'va-benefits/basic-information',
+          title: 'Current Compensation',
```

Comment:
Page titles should be sentence case. Chapter titles are title case, so thank you for updating those.